src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/document/output/Pdf.java

```diff
@@ -20,4 +23,24 @@ public String getFilename() {
     public byte[] getBytes() {
         return bytes;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Pdf pdf = (Pdf) o;
+        return Objects.equals(fileBaseName, pdf.fileBaseName) && Arrays.equals(bytes, pdf.bytes);
+    }
+
+    @Override
+    public int hashCode() {
+
+        int result = Objects.hash(fileBaseName);
+        result = 31 * result + Arrays.hashCode(bytes);
+        return result;
+    }
```

Comment:
are those actually used? (you can compare objects with assertj methods in tests)

Reply:
Yes by the mockito verifications.